### DISCUSSION TITLE
希望能增加游戏登录方式的可选项

### DISCUSSION THREAD

**Discussion by ying727** [Meloong-Git/PCL#2043] [讨论]
在PCL2中，默认的服务器登录方式为正版登录或离线登录，在首页可快捷选择使用正版或离线账号进入游戏，但没有通过Authlib-Injector实现的第三方登录和正版登录的可选项，给使用上带来了一点不便，希望能增加不同登录方式的组合。且希望服务器登录方式能做成一个全局设置，能在启动器设置内填写第三方服务器登录信息和选择服务器登录方式
![image](https://github.com/Hex-Dragon/PCL2/assets/61232882/230087ca-9657-409f-84e3-a162f7e4bcb2)

![image](https://github.com/Hex-Dragon/PCL2/assets/61232882/9dd09489-9fde-4cf9-9ac2-98bd482bcea6)

**Comment by Chiloven945** [comment]
请于 #58 为其投票
还有就是请不要乱交Categories……讨论是给讨论事情用的不是提建议，提建议请去issues
